core/src/main/java/org/mskcc/cbio/portal/dao/DaoGeneOptimized.java

```diff
@@ -169,18 +169,13 @@ public int addGene(CanonicalGene gene) throws DaoException {
     }
     
     /**
-     * Update database with gene length
-     * @return number of records updated.
-     * @throws DaoException 
+     * Update Gene Record in the Database. It will also replace this 
+     * gene's aliases with the ones found in the given gene object.
```

Comment:
Does it the function still return the number of records updated? If so, is there a reason not to document this?